hack/verify-deps.sh

```diff
@@ -24,7 +24,7 @@ _tmpdir="$(mktemp -d -t verify-deps.XXXXXX)"
 cd "${_tmpdir}"
 _tmpdir="$(pwd)"
 
-trap "rm -rf ${_tmpdir}" EXIT
+trap 'rm -rf ${_tmpdir}' EXIT
```

Comment:
shellcheck seems to not catch this, but should we be quoting the `${}` here like `'rm -rf "${_tmpdir}"'`?